### PR TITLE
Some changes to get working on OSX

### PR DESCRIPTION
**Pull request by mi-yu**

### CODE REVIEW

File: examtex [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 # save current directory
 cwd=$(pwd)
 # change directory to source
-cd "$(dirname "$(readlink -f "$0")")"
+cd "$(dirname "$(greadlink -f "$0")")"
```
Comment (mi-yu) [inline examtex]: OSX `readlink` doesn't have the `-f` flag :( maybe you can do some detection or something but also not everyone has `greadlink`

File: examtex [hunk 2]
```diff
@@ -37,11 +37,11 @@ if [ "$err" != "" ]; then
 	echo "$err"
 # compile pdf from tex
 elif [ $to_pdf = 'true' -a "$err" = "" ]; then
-	cd "$(dirname "$1")"
-	latexmk -pdf -quiet "${1%.*}-EXAM.tex"
-	latexmk -pdf -quiet "${1%.*}-KEY.tex"
+	cd $cwd
+	latexmk -pdf -f -quiet "${1%.*}-EXAM.tex"
```
Comment (mi-yu) [inline examtex]: was getting weird build errors and it wouldn't complete unless i had this `-f` flag

**Review by mi-yu — COMMENTED** [review]
also was missing some packages (exam, siunitx, etc), maybe there is a way to specify them in a requirements file and download in one command

File: examtex (hunk 2 above)
Comment (dkarkada) [reply examtex]: try compiling to tex and then using sublime to compile to pdf. Tryna see if the program is generating wrong tex files..

**Comment by dkarkada** [comment]
hopefully changing to python will make it more cross-platform!! we love python!!

**Pull request closed without merging**